modules/apps/portal-osgi-debug/portal-osgi-debug-impl/src/main/java/com/liferay/portal/osgi/debug/internal/osgi/commands/SystemCheckOSGiCommands.java

```diff
@@ -61,10 +63,24 @@ protected void activate(BundleContext bundleContext) {
 					"the command \"system:check\" in Gogo shell.");
 		}
 
-		if (GetterUtil.getBoolean(
-				bundleContext.getProperty("initial.system.check.enabled"),
-				true)) {
+		boolean checkEnabled = true;
+		ServiceReference<Props> serviceReference =
+			bundleContext.getServiceReference(Props.class);
 
+		if (serviceReference != null) {
```

Comment:
@W2jx `SystemCheckOSGiCommands` is a `@Component` so you can just `@Reference Props _props` and use it.